livekit-agents/livekit/agents/ipc/proc_pool.py

```diff
@@ -66,7 +66,7 @@ def get_by_job_id(self, job_id: str) -> JobExecutor | None:
             None,
         )
 
-    def get_running_jobs(self) -> list[RunningJobInfo]:
```

Comment:
lol what I mean is that this function is already in the worker

Reply:
ohhhh lol removed